xds/src/main/java/io/grpc/xds/VirtualHost.java

```diff
@@ -45,14 +45,14 @@
   // The list of routes that will be matched, in order, for incoming requests.
   abstract ImmutableList<Route> routes();
 
-  @Nullable
   abstract ImmutableMap<String, FilterConfig> filterConfigOverrides();
 
   public static VirtualHost create(
       String name, List<String> domains, List<Route> routes,
       Map<String, FilterConfig> filterConfigOverrides) {
     return new AutoValue_VirtualHost(name, ImmutableList.copyOf(domains),
-        ImmutableList.copyOf(routes), ImmutableMap.copyOf(filterConfigOverrides));
+        ImmutableList.copyOf(routes),
+        ImmutableMap.copyOf(checkNotNull(filterConfigOverrides, "filterConfigOverrides")));
```

Comment:
The `checkNotNull` here is useless/redundant. If the argument is null, it breaks immediately at the same line.

AutoValue is very strict for nullness of properties. If the property is annotated with `@Nullable` (just as what it was), the (generated) constructor skips null check for it. Otherwise, every property is checked inside the constructor and you don't need to do it yourself.